cmd/cmd/remote.go

```diff
@@ -39,6 +39,8 @@ Available Commands:
   update [nick-name] [host] [port]         Update a Micro Integrator
   select [nick-name]                       Select a Micro Integrator on which commands are executed
   show                                     Show available Micro Integrators
+  login                                    Login to the current Micro Integrator instance
```

Comment:
"Login to the selected Micro Integrator"?

Reply:
Fixed in 2514a25d799cfe90b35dd0fdda17773b127eb19c